.gitignore

```diff
@@ -43,4 +43,4 @@ grafast/dataplan-pg/__tests__/**/*.mermaid.png
 /Session.vim
 /WHAT_ARE_YOU_DOING.md
 contrib
-/env
+/env
```

Comment:
I think you've removed the newline from the end of `.gitignore` - perhaps when you removed the nix stuff?

Reply:
I'll put it back, :-)